app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -192,14 +192,27 @@ def container_conditions_query_for(relation)
   def container_conditions_for(relation)
     # TODO: what if last parent disappears, will this ||= never happen
     #   and the ContainerConditions won't be deleted?
-    @inv_collections[[:container_conditions_for, relation.model]] ||= ::ManagerRefresh::InventoryCollection.new(
+    @inv_collections[[:container_conditions_for, relation.model.name]] ||= ::ManagerRefresh::InventoryCollection.new(
```

Comment:
it will be probably better to use only symbols as indexes, e.g.

@inv_collections["container_conditions_for_#{relation.model.name.underscore}".to_sym], so these can be then used as a method calls, when we move the definitions to the Persister

persister.container_conditions_for_pod.build(data)
persister.to_yaml